### ISSUE TITLE
Option to equip golden pan/saxxy/golden wrench

### ISSUE THREAD

**Issue by ghost** [punteroo/TF2-Item-Plugins#4]
This option from vip-australiums was removed with the creation of tf2items_weapons, I'm wondering if this will come back, or perhaps a separate plugin could be made specifically for equipping items players don't have.

**Comment by punteroo** [comment]
Since the project's porting to TFEconData was mostly because of offering performance and deleting usage of TF2IDB, giving items the player does not have wasn't something I anticipated to release first.
But yeah, I'm going to include this functionality again for **tf2item_weapons** on the next issued release. I wanted to release the _main functionality_ first before going for something fully fledged and complete (because I took a long time to finish the port :P)

TL;DR: Functionality will be included on the next release, as I wanted to get out the main port first because of the long time I took to finish it.

**Comment by punteroo** [comment]
This was now implemented on the latest release v3.0.1. Thank you!

https://github.com/punteroo/TF2-Item-Plugins/releases/tag/v3.0.1
